src/dispnew.c

```diff
@@ -5956,35 +5956,6 @@ init_display (void)
 
 
 
-/***********************************************************************
-			   Blinking cursor
- ***********************************************************************/
-
-DEFUN ("internal-show-cursor", Finternal_show_cursor,
-       Sinternal_show_cursor, 2, 2, 0,
-       doc: /* Set the cursor-visibility flag of WINDOW to SHOW.
-WINDOW nil means use the selected window.  SHOW non-nil means
-show a cursor in WINDOW in the next redisplay.  SHOW nil means
-don't show a cursor.  */)
-  (Lisp_Object window, Lisp_Object show)
-{
-  /* Don't change cursor state while redisplaying.  This could confuse
-     output routines.  */
```

Comment:
Why drop the comment ?

Reply:
Sorry, forgot to copy those over.

Reply:
I wrote https://gist.github.com/shaleh/c84d905b0bc755163beea8269be00f1d to help. `M-| rustize-emacs`. It needs some tweaks to catch all of the C code styles but it is a start.